### PR TITLE
share a folder with some resources in it

### PR DESCRIPTION
**Pull request by jasson99**
## Description
This pr adjusts the public link sharing test with uploader role, such that a folder having some resources 

## Related Issue
- Fixes https://github.com/owncloud/ocis/issues/723

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by individual-it — APPROVED** [review]

**Comment by phil-davis** [comment]
The line numbers in the feature file changed, so expected-failures needs to be adjusted. https://drone.owncloud.com/owncloud/web/12873/78/11

**Comment by phil-davis** [comment]
Also, please make sure to rebase to the very latest `master` because there have been changes to `run.sh` in master today.

File: tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md [hunk 1]
```diff
@@ -273,11 +273,11 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:312](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L311)
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:320](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L319)
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:329](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L328)
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:338](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L337)
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:347](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L346)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md:280]:
```suggestion
-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:312](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L312)
-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:320](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L320)
-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:329](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L329)
-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:338](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L338)
-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:347](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L347)
```

The line numbers need to change in 2 places.

File: tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md [hunk 2]
```diff
@@ -412,10 +412,10 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
--   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:212](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L212)
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:123](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L122)
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:299](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L298)
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:276](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L275)
+-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:213](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L212)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md:418]:
```suggestion
-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:123](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L123)
-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:299](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L299)
-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:276](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L276)
-   [webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:213](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature#L213)
```

**Review by phil-davis — APPROVED** [review]

**Pull request merged**